packages/plugins/bundler-report/src/index.ts

```diff
@@ -96,12 +88,14 @@ export const getBundlerReportPlugins = (context: GlobalContext): PluginOptions[]
                 }
 
                 if ('output' in options) {
-                    handleOutputOptions(options.output);
+                    const outputOptions = Array.isArray(options.output)
+                        ? options.output
+                        : [options.output];
+                    for (const output of outputOptions) {
+                        handleOutputOptions(output);
+                    }
                 }
             },
-            outputOptions(options) {
-                handleOutputOptions(options);
```

Comment:
This (and above) resulted in a double execution.

It seems that `outputOptions` is just a hook called for EACH output option (it can be an array) passed to the configuration.

Where `options` is the "raw" full option object.